testing.md

```diff
@@ -0,0 +1,32 @@
+# Golden Test Data
+
+## Overview
+
+The data/test directory contains the inputs and expected outputs used to determine that everything works as expected when pulling in the latest version of the L2-L3 code.
+
+## How to run
+
+#TODO: ## Automate the running of these tests
+#TODO: ## Generate and compare netcdf outputs
+
+Run the following steps from the root directory of this github repo
+
+1. Make sure you have the latest version of the test data
+
+    ```
+    dvc update
+    ```
+
+1. conda activate level3
```

Comment:
Is there another place where we tell someone how to create the `level3` environment? If not can you add it here?

Reply:
Done.